virt/getting_started/virt-using-the-cli-tools.adoc

```diff
@@ -28,3 +28,7 @@ include::modules/virt-deploying-libguestfs-with-virtctl.adoc[leveloffset=+1]
 
 include::modules/virt-about-libguestfs-tools-virtctl-guestfs.adoc[leveloffset=+2]
 
+[id="using_ansible"]
+== Using Ansible
+To use the Ansible collection for {VirtProductName}, see link:https://console.redhat.com/ansible/automation-hub/repo/published/redhat/openshift_virtualization[Red Hat Ansible Automation Hub].
```

Comment:
```suggestion
To use the Ansible collection for {VirtProductName}, see link:https://console.redhat.com/ansible/automation-hub/repo/published/redhat/openshift_virtualization[Red Hat Ansible Automation Hub] ({hybrid-console})
```

Reply:
This will further signify to our customers where they're going to visit in advance of clicking the link.

Reply:
@dfitzmau  Done, thank you